packages/mongo-nav/src/mongo-select/OrgSelect.tsx

```diff
@@ -200,6 +206,30 @@ function OrgSelect({
     }
   };
 
+  function fetchOrgsAsGlobalUser(searchTerm: string) {
+    const queryString = searchTerm ? `?term=${searchTerm}` : '';
+    const endpointURI = `${hosts.cloud}/user/shared/organizations/search${queryString}`;
+    return fetch(endpointURI, {
+      credentials: 'include',
+      mode: 'cors',
+      method: 'GET',
+    });
+  }
+
+  const filterDataAsAdmin = async () => {
+    const response = await fetchOrgsAsGlobalUser(value);
```

Comment:
I looked into this and found a set of eslint rules by Salesforce that include one for async/await. It seems like it has more detail than we would want, but I just wanted to throw it out there in case we want to add a custom rule to prevent async/await:

https://github.com/salesforce/eslint-plugin-lwc/blob/master/lib/rules/no-async-await.js